lib/beaker/host/windows.rb

```diff
@@ -38,11 +38,11 @@ def determine_ssh_server
       return @ssh_server if @ssh_server
       @ssh_server = :openssh
       status = execute('cmd.exe /c sc query BvSshServer', :accept_all_exit_codes => true)
-      if /4  RUNNING/.match?(status)
+      if status.include?('4  RUNNING')
         @ssh_server = :bitvise
       else
         status = execute('cmd.exe /c sc qc sshd', :accept_all_exit_codes => true)
-        if /C:\\Windows\\System32\\OpenSSH\\sshd\.exe/.match?(status)
+        if status.include?('C:\\Windows\\System32\\OpenSSH\\sshd.exe')
```

Comment:
I think you need to unescape the `\` if you're using single quotes
```suggestion
        if status.include?('C:\Windows\System32\OpenSSH\sshd.exe')
```

Reply:
`\` in single quoted string allows to escape a `'`:

```ruby
'Hello \'world\'!' == "Hello 'world'!" #=> true
```

A `\` to escape a non special char will output a literal `\` before that char, so it feels more natural to me to use `\\`:

```ruby
'C:\Windows'  #=> "C:\\Windows"
'C:\\Windows' #=> "C:\\Windows"
```

TL;DR: I would keep the line as it is.